<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCopyModelsFromOtherSwiftVersionsComponent</class>
 <widget class="QFrame" name="CCopyModelsFromOtherSwiftVersionsComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy models from other swift versions</string>
  </property>
  <layout class="QVBoxLayout" name="vl_CopyModelsFromOtherSwiftVersions">
   <item>
    <widget class="QGroupBox" name="gb_OtherSwiftVersions">
     <property name="title">
      <string>Select the version to copy from</string>
     </property>
     <layout class="QVBoxLayout" name="vl_OtherSwiftVersions">
      <item>
       <widget class="swift::gui::components::COtherSwiftVersionsComponent" name="comp_OtherSwiftVersions">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>125</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_WhatToCopy">
     <property name="title">
      <string>Select what to copy</string>
     </property>
     <layout class="QHBoxLayout" name="hl_WhatToCopy">
      <item>
       <widget class="QCheckBox" name="cb_ModelSet">
        <property name="text">
         <string>Model set</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_ModelCache">
        <property name="text">
         <string>Model cache</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_Of">
        <property name="text">
         <string>simulator:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::components::CSimulatorSelector" name="comp_SimulatorSelector">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_SelectWhatToCopy">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Silent">
        <property name="text">
         <string>silent (no dialogs)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_StartCopying">
        <property name="text">
         <string> start copy </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Progress">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Progress">
      <item>
       <widget class="QFrame" name="fr_Status">
        <layout class="QFormLayout" name="fl_Status">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_Status">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_Status">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::views::CAircraftModelView" name="tvp_AircraftModels">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::COtherSwiftVersionsComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/otherswiftversionscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::views::CAircraftModelView</class>
   <extends>QTableView</extends>
   <header>gui/views/aircraftmodelview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cb_ModelSet</tabstop>
  <tabstop>cb_ModelCache</tabstop>
  <tabstop>cb_Silent</tabstop>
  <tabstop>pb_StartCopying</tabstop>
  <tabstop>le_Status</tabstop>
  <tabstop>tvp_AircraftModels</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
